src/index.ts

```diff
@@ -1,9 +1,26 @@
 #!/usr/bin/env node
 
 import yargs from "yargs"
+import { TeamsMessaging } from "./teams/teams-messaging";
+
+
+const args = require("yargs").usage("jscm command")
+    .command( 'teams','This command is used to send a teams message via a passed webhook',
```

Comment:
formatting is **very** off here. have you installed https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode ?